docs/asciidoc/processors.adoc

```diff
@@ -45,7 +45,7 @@ If, after exploring the above alternatives, you still think you need a `Processo
 the <<processor-overview>> section to learn about the different implementations.
 
 [[sinks]]
-= Safely Produce from Multiple Threads by Using `StandaloneFluxSink` and `StandaloneMonoSink`
+= Safely Produce from Multiple Threads by Using `Sink` and `Sink`
```

Comment:
IF the mono flavor is removed, this `and 'Sink'` can be removed. but I'm not a huge fan of removing the mono flavor...

Reply:
I think that is something I'm not too attached to personally if you think having a dedicated mono is valuable. I'll clean the doc in the final pass. The main problem I'm attacking is removing Processors which is more impacting IMO.